### ISSUE TITLE
add component for looking up a user-supplied word

### ISSUE THREAD

**Issue by balmas** [alpheios-project/components#10]
we need a component that can be used to look up a user-supplied word. This probably would be added to the panel somewhere

**Comment by irina060981** [comment]
I have created a separate branch - lookup-component
https://github.com/alpheios-project/components/tree/lookup-component

The main points:

1) a separate component - lookup.vue and put it in popup.vue (it seems to me to be more comfortable 
    for using there, but iy could be moved to another place)
    it looks like this
    ![image](https://user-images.githubusercontent.com/12377640/39567332-0326bb4e-4f02-11e8-8e35-c60c4262d392.png)

2) lookup.vue component needs two properties: uiController and data.l10n.messages for rendering the lexical query - a popup.vue has those properties

3) I have created a subclass LexicalQueryLookup for getting data, because it needs to initiate a query with initiating data adapters. I created it separately to be more flexiable.

4) I have added an additional function to text selector to be created from a simple string (not HTML Selector)

5) Also as I started from master branch there were not some previous fixes from lemma-translation, I have repeated them and I have corrected unit testing according to fixes of lemma-translation.

I have created a Pull Request
https://github.com/alpheios-project/components/pull/39
